src/VaultV2.sol

```diff
@@ -229,6 +232,7 @@ contract VaultV2 is IVaultV2 {
 
     mapping(bytes4 selector => uint256) public timelock;
     mapping(bytes data => uint256) public executableAt;
+    mapping(bytes4 selector => uint256) public pendingCount;
```

Comment:
also works, not sure
```suggestion
    mapping(bytes4 selector => uint256) public pending;
```

Reply:
slightly less clear, I prefer the longer one